### ISSUE TITLE
aurelia-configuration.d.ts should define a module

### ISSUE THREAD

**Issue by dkent600** [Vheissu/Aurelia-Configuration#57]
OK, I'm a newbie at typescript and typings, but I have not been able to import aurelia-configuration as the d.ts files are written.  The crux seems to be that it doesn't define a module.

1)  First, though perhaps not relevant, I renamed it index.d.ts, for consistency with every other typings file I've ever seen

2) second, I merged the contents of the configure.d.ts into the newly-renamed index.d.ts as follows, and it works:

```
declare module 'aurelia-configuration' {

import { Loader } from 'aurelia-loader';

interface ConfigInterface {
}

class Configure {
    loader: Loader;
    environment: string;
    environments: any;
    directory: string;
    config_file: string;
    cascade_mode: boolean;
    private _config_object;
    private _config_merge_object;
    constructor(loader: Loader);
    setDirectory(path: any): void;
    setConfig(name: any): void;
    setEnvironment(environment: any): void;
    setEnvironments(environments?: boolean): void;
    setCascadeMode(bool?: boolean): void;
    obj: ConfigInterface;
    config: string;
    is(environment: any): boolean;
    check(): void;
    environmentEnabled(): boolean;
    environmentExists(): boolean;
    get(key: any, defaultValue?: any): any;
    set(key: any, val: any): void;
    merge(obj: any): void;
    lazyMerge(obj: any): void;
    setAll(obj: any): void;
    getAll(): ConfigInterface;
    loadConfig(): Promise<void>;
    loadConfigFile(path: any, action: any): Promise<void>;
    mergeConfigFile(path: any): Promise<void>;
}

    export function configure(aurelia: any, configCallback: any): Promise<{}>;

    export { Configure };
}
```

**Comment by mikeesouth** [comment]
@dkent600 I didn't have to rename the file (it was already called index.d.ts in the release I had/have) but I also got an error and I had to change the content to the same as you have. @Vheissu you did a similar change in commit d41896c301cb796ce0a07cd06046c18a86b27bd5 that is included in the 1.0.12 release but unfortunately it does not work for me. With release 1.0.12 I get this error:
```
[default] C:\projects\foo\src\bar.ts:5:25
    File 'C:/projects/foo/node_modules/aurelia-configuration/dist/index.d.ts' is not a module.
[default] Checking finished with 1 errors
```
When changing the dist/index.d.ts file to the contents that @dkent600 provided above, then it works.
